### ISSUE TITLE
Add alt text to plots and images

### ISSUE THREAD

**Issue by teunbrand** [tidyverse/ggplot2#5222]
As part of the upkeep #5220, alt-text should be added to plots. The details for adding alt text are explained in https://posit.co/blog/knitr-fig-alt/.

I presume this mostly applies to vignettes and articles, but if other documents should benefit from alt text, it would be good to mention them here.
